diem-unified-charter.md

```diff
@@ -26,12 +26,10 @@ Which attributes an emblem contains, and how a digital emblem is secured and pre
 
 # Initial Scope
 
-The DIEM WG will ensure maximum reuse of work for emblem representation and validation, facilitate re-use of existing protocols and capabilities, and ensure that existing standards are leveraged appropriately.
-A discovery mechanism for the initial work will only be specified by this group after the initial emblem validation procedure is completed (see the Deliverables section below).
+The DIEM WG will develop digital emblems that explicitly identify their bearer. It will also develop an initial DNS-based discovery mechanism.
+In particular, the design of discovery mechanisms using proximity-based protocols such as QRCodes, NFC, or Bluetooth is out-of-scope.
+The discovery mechanism, presentation, and validation procedures must be designed so that validators using them remain undetectable as such.
```

Comment:
```suggestion
The discovery mechanism, presentation, and validation procedures must not preclude the use of means so that validators using them are not detectable as such.
```

Reply:
> The discovery mechanism, presentation, and validation procedures must allow for validators using them remaining undetectable as such.